### PR TITLE
Support clusters running Istio

### PR DESCRIPTION
**Pull request by eyalb181**

### CODE REVIEW

**Comment by t4lz** [comment]
Changelog? :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ## [Unreleased]
 
+### Fixed
+- Support clusters running Istio. Closes [[[#485](https://github.com/metalbear-co/mirrord/issues/485)]].
```
Comment (t4lz) [inline CHANGELOG.md:11]:
```suggestion
- Support clusters running Istio. Closes [[#485](https://github.com/metalbear-co/mirrord/issues/485)].
```

**Comment by t4lz** [comment]
`cargo fmt --all`

**Comment by aviramha** [comment]
I think the distinguishing of linekrd/istio is futile. I would change the name of linkerd variant to mesh and have a const hashset that is checked against for mesh.

**Review by aviramha — APPROVED** [review]

**Pull request merged**